File: Algorithms/src/andy/javascript/level1/minimumrectangle.js
// 프로그래머스 Level 1,최소직사각형
function solution(sizes) {
    var func = { };
    func.sz = 1000000;    
    var arrx = [];
    var arry = [];
    sizes.forEach( (d) => {
        arrx.push(Math.max(d[0],d[1]));
        arry.push(Math.min(d[0],d[1]));
    });
    
    return Math.max(...arrx) * Math.max(...arry);
}

File: Algorithms/src/andy/level1/minimumrectangle.java
package andy.level1;


// 프로그래머스 Level 1,최소직사각형
public class minimumrectangle {
    public int solution(int[][] sizes) {
        int x=0,y=0;
        for(int []d:sizes) {
            if(d[0]>d[1]) {
                x = Math.max(x, d[0]);
                y = Math.max(y, d[1]);
            }
            else {
                x = Math.max(x, d[1]);
                y = Math.max(y, d[0]);
            }
        }
        return x*y;
    }
}
